parl/utils/machine_info.py

```diff
@@ -121,23 +125,26 @@ def is_xpu_available():
       True if paddle was complied with XPU.
     """
     xpu_count = int(os.getenv("FLAGS_selected_xpus", "-1"))
-    ret = xpu_count >= 0
+    if xpu_count < 0:
+        logger.info(
+            'Cannot find available XPU devices, using other devices now.')
```

Comment:
Do not print this message since not all the users know XPU. They may concern if they run the code uncorrectly.